thinc/backends/_custom_kernels.py

```diff
@@ -519,7 +533,7 @@ def backprop_reduce_sum(d_sums, lengths, *, threads_per_block=128, num_blocks=12
     O = d_sums.shape[1]
     _check_lengths(lengths, T)
 
-    out = cupy.zeros((T, O), dtype=d_sums.dtype)
+    out = _alloc((T, O), dtype=d_sums.dtype, zeros=True)
```

Comment:
`backprop_reduce_mean` should backprop to every sequence item, so I think this can be `zeros=False`.